concepticondata/conceptlists/Carvalho-2021-206.tsv

```diff
@@ -0,0 +1,207 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Carvalho-2021-206-1	1	body	1480	BODY
+Carvalho-2021-206-2	2	head	1256	HEAD
+Carvalho-2021-206-3	3	eye	1248	EYE
+Carvalho-2021-206-4	4	earhole	3891	EARHOLE
+Carvalho-2021-206-5	5	hair/feather	2381	FEATHER OR FUR OR HAIR
+Carvalho-2021-206-6	6	mouth	674	MOUTH
+Carvalho-2021-206-7	7	lip	478	LIP
+Carvalho-2021-206-8	8	beard	678	BEARD
+Carvalho-2021-206-9	9	tongue	1205	TONGUE
+Carvalho-2021-206-10	10	nose	1221	NOSE
+Carvalho-2021-206-11	11	bone	1394	BONE
+Carvalho-2021-206-12	12	back	1291	BACK
+Carvalho-2021-206-13	13	breast	1402	BREAST
+Carvalho-2021-206-14	14	breast	796	NIPPLE
+Carvalho-2021-206-15	15	chest	1592	CHEST
+Carvalho-2021-206-16	16	heart	1223	HEART
+Carvalho-2021-206-17	17	liver	1224	LIVER
+Carvalho-2021-206-18	18	blood	946	BLOOD
+Carvalho-2021-206-19	19	flesh	2259	FLESH
+Carvalho-2021-206-20	20	Veins	1924	VEIN
+Carvalho-2021-206-21	21	buttocks	834	BUTTOCKS
+Carvalho-2021-206-22	22	leg	1297	LEG
+Carvalho-2021-206-23	23	thigh	800	THIGH
+Carvalho-2021-206-24	24	knee	1371	KNEE
+Carvalho-2021-206-25	25	shin	186	SHIN
+Carvalho-2021-206-26	26	foot	1301	FOOT
+Carvalho-2021-206-27	27	arm	1673	ARM
+Carvalho-2021-206-28	28	elbow	981	ELBOW
+Carvalho-2021-206-29	29	hand	1277	HAND
+Carvalho-2021-206-30	30	neck	1333	NECK
+Carvalho-2021-206-31	31	skin	763	SKIN
+Carvalho-2021-206-32	32	navel	1838	NAVEL
+Carvalho-2021-206-33	33	tail	1220	TAIL
+Carvalho-2021-206-34	34	wing	1257	WING
+Carvalho-2021-206-35	35	egg	744	EGG
+Carvalho-2021-206-36	36	feces	675	FAECES (EXCREMENT)
+Carvalho-2021-206-37	37	fart	1159	FART
+Carvalho-2021-206-38	38	urine	1142	URINE
+Carvalho-2021-206-39	39	fat	323	FAT (ORGANIC SUBSTANCE)
+Carvalho-2021-206-40	40	name	1405	NAME
+Carvalho-2021-206-41	41	house	1252	HOUSE
+Carvalho-2021-206-42	42	village	930	VILLAGE
+Carvalho-2021-206-43	43	nest	539	NEST
+Carvalho-2021-206-44	44	path	2252	PATH
+Carvalho-2021-206-45	45	path2	2252	PATH
```

Comment:
What's up with these two paths? And the two fathers and mothers ancd so on below?

Reply:
you're right, this list is basically a listing of cognate sets and the duplicate glosses refer to different cognates, respectively. The data for this list was gathered from a vast number of dictionaries and grammars on the target languages so it definitely wasn't designed for elicitation.